### DISCUSSION TITLE
2021 年终总结 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#1] [General]
# 2021 年终总结 | 高策

江湖小虾米

http://gaocegege.com/Blog/%E9%9A%8F%E7%AC%94/newyear2021

**Comment by linxuyalun** [comment]
> 我似乎渐渐能感受到我和年轻人之间的差异了

我今年收到的最大暴击就是我组会上随口问了个本科生是九几年的，然后他说他零一年的

**Reply by gaocegege** [reply]
蚌埠住了
